studies/us_states_validation/etl.py

```diff
@@ -32,26 +32,79 @@ def import_and_clean_cases(save_path: Path) -> pd.DataFrame:
     return df
 
 
-def get_rt_live_data(save_path: Path) -> pd.DataFrame:
-    '''
-    Gets Rt estimates from Rt.live.
-    '''
+def get_adaptive_estimates(path: Path) -> pd.DataFrame:
+    
+    # Parameters for filtering raw df
+    kept_columns   = ['date','state','RR_pred','RR_CI_lower','RR_CI_upper','T_pred',
+                      'T_CI_lower','T_CI_upper','new_cases_ts','anamoly']
```

Comment:
i hate being a pedant but `anomaly`

Reply:
Oof yeah, I kept messing up that spelling. Good catch.